README.rst

```diff
@@ -349,8 +349,9 @@ Release Notes
 User Facing
 ~~~~~~~~~~~
 
-- Added support Python 3.3 and 3.4. [PR `#38`_]
-- Bumped up the minimum dependencies for NumPy, SciPy, and Cython.
+- Dropped support for IPython < 3.0. [PR `#237`_]
+- Added support Python 3.3 and 3.4. [PR `#229`_]
```

Comment:
I rebased and then changed it here to point to the PR instead of the original issue to follow the same pattern as the others.